keps/sig-node/4381-dra-structured-parameters/README.md

```diff
@@ -532,44 +530,47 @@ reconstructed by the driver. This has no effect on already allocated claims
 because the allocation result is tracked in those claims, not the
 `ResourceSlice` objects (see [below](#state-and-communication)).
 
-#### Devices as a named list of attributes
+#### Devices as a named list
 
-Embedded inside each `ResourceSlice` is a list of one or more devices, represented as a named list of attributes.
+Embedded inside each `ResourceSlice` is a list of one or more devices, each with sets of attributes
+and quantities.
 
 ```yaml
 kind: ResourceSlice
 apiVersion: resource.k8s.io/v1alpha3
 ...
 spec:
   # The node name indicates the node.
+  #
   # Each driver on a node provides pools of devices for allocation,
-  # with unique device names inside each pool.
-  # Usually, but not necessarily, that pool name is the same as the
-  # node name.
+  # with unique device names inside each pool. A pool might be defined
+  # by more than one ResourceSlice.
   nodeName: worker-1
-  poolName: worker-1
-  driverName: cards.dra.example.com
+  pool:
+    ...
```

Comment:
Why not populate this for a more complete example?

Reply:
I think it's irrelevant at this early stage.

Tim went further in one of his review comments and didn't want to start the KEP with YAML code at all. The argument was that this forces people to learn about the API instead of getting to know concepts first.

I'm not going that far yet in this PR, but not showing a pool is one step in that direction.

Reply:
That's fair. Let's leave it as is then.